Ramda/Extensions/empty.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  empty.swift
+//  Ramda
+//
+//  Created by Christian Leovido on 27/09/2019.
+//
+
+import Foundation
+
+public protocol EmptyDefinable {
+
+    associatedtype CollectionType: Collection
+    associatedtype SubCollectionType: Collection
```

Comment:
Whats the purpose to the `SubCollectionType`?

Reply:
No purpose at all. I removed it